README.md

```diff
@@ -13,6 +13,7 @@ The goal of this tutorial is to demonstrate how to get started with an integrati
 Please see the [Introduction](https://github.com/Watson-Explorer/wex-wdc-integration-samples) for an overview of the integration architecture, and the tools and libraries that need to be installed to create Java-based applications in Bluemix.
 
 - An [IBM Bluemix account](https://bluemix.net/)
+- [Cloud Foundry](https://github.com/cloudfoundry/cli/releases)
```

Comment:
Cloud foundry is required for all of the WEX-WDC integration examples, and so is included in the list of prereqs discussed at the "Introduction" link above.  If we list Cloud foundry here, we should also list Apache Maven and the IBM JDK.